### PR TITLE
better watching

### PR DESCRIPTION
**Pull request by martinraison**
@2is10 using "stream mode" of gulp-watch instead of "batch mode".

### CODE REVIEW

File: packrat/gulpfile.js [hunk 1]
```diff
@@ -337,15 +343,14 @@ function watchAndReload(target) {
...
-      .pipe(watch(function(files) {
-        var withLazypipe = pipes.map(function (pipe) {
-          return files.pipe(pipe());
-        });
-        return es.merge.apply(null, withLazypipe)
-          .pipe(es.wait(reloader()));
-      }));
+  if (pipes.length > 0) {
+    var watchers = pipes.map(function (pipe) {
+      return watch({glob:target, base: './', emitOnGlob: false})
```
Comment (martinraison) [inline packrat/gulpfile.js:348]: `emitOnGlob: false` prevents files from being processed twice on startup

File: packrat/gulpfile.js [hunk 2]
```diff
@@ -363,6 +368,7 @@ gulp.task('watch', function() {
 });
 
 gulp.task('package', function () {
+  isRelease = true;
```
Comment (2is10) [inline packrat/gulpfile.js:371]: Part of me cringes at the use of global state. Is there no way to parameterize tasks?

**Comment by 2is10** [comment]
Nice. Thanks.

File: packrat/gulpfile.js (hunk 2 above)
Comment (martinraison) [inline packrat/gulpfile.js:371]: cf discussion (and for a bit more context: https://github.com/orchestrator/orchestrator/issues/17)

**Pull request merged**